test/send.spec.ts

```diff
@@ -157,22 +228,62 @@ describe("Send to Queue/Subscription", function(): void {
     await testSimpleSend(partitionedQueueClient, partitionedQueueClient);
   });
 
-  it("Simple send using Topics and Subscriptions", async function(): Promise<void> {
+  it("Simple send using Partitioned Topics and Subscriptions", async function(): Promise<void> {
     await testSimpleSend(partitionedTopicClient, partitionedSubscriptionClient);
   });
 
-  it("Simple send using UnPartitioned Queues", async function(): Promise<void> {
+  it("Simple send using unPartitioned Queues", async function(): Promise<void> {
     await testSimpleSend(unpartitionedQueueClient, unpartitionedQueueClient);
   });
 
-  it("Simple send using UnPartitioned Topics and Subscriptions", async function(): Promise<void> {
+  it("Simple send using unPartitioned Topics and Subscriptions", async function(): Promise<void> {
     await testSimpleSend(unpartitionedTopicClient, unpartitionedSubscriptionClient);
   });
 
+  it("Simple send using Partitioned Queues with Sessions", async function(): Promise<void> {
+    await testSimpleSend(partitionedQueueSessionClient, partitionedQueueMessageSession, true);
+  });
+
+  it("Simple send using Partitioned Topics and Subscriptions with Sessions", async function(): Promise<
+    void
+  > {
+    await testSimpleSend(
+      partitionedTopicSessionClient,
+      partitionedSubscriptionMessageSession,
+      true
+    );
+  });
+
+  it.only("Simple send using unPartitioned Queues with Sessions", async function(): Promise<void> {
+    await testSimpleSend(unpartitionedQueueSessionClient, unpartitionedQueueMessageSession, true);
+  });
+
+  it.only("Simple send using unPartitioned Topics and Subscriptions with Sessions", async function(): Promise<
```

Comment:
Why are we adding "Only" here and in other "it" and "describe" section?

Reply:
We dont need that, I forgot to remove it after testing that test